src/status/profile/profile.nim

```diff
@@ -51,3 +53,8 @@ proc toProfileModel*(profile: JsonNode): Profile =
   
   if profile.hasKey("localNickname"):
     result.localNickname = profile["localNickname"].str
+  if profile.hasKey("images"):
+    if profile["images"].hasKey("thumbnail"):
+      result.identityImage.thumbnail = profile["images"]["thumbnail"]["uri"].str
+    if profile["images"].hasKey("large"):
+      result.identityImage.large = profile["images"]["large"]["uri"].str
```

Comment:
I think this can be simplified like this:
```suggestion
  result.identityImage.thumbnail = profile{"images"}{"thumbnail"}{"uri"}.getStr()
  result.identityImage.large = profile{"images"}{"large"}{"uri"}.getStr()
```

Reply:
I tried something like this and it failed, but I might not have done it exactly like that